packages/app/src/embed/components/File/index.js

```diff
@@ -29,13 +29,16 @@ export default class File extends React.PureComponent<Props> {
   };
 
   render() {
-    const { title, depth, type, active, alternative } = this.props;
+    const { title, depth, type, active, alternative, onClick } = this.props;
     return (
       <div>
         <Entry
           alternative={alternative}
           active={active}
-          onClick={this.setCurrentModule}
+          onClick={() => {
+            onClick && onClick();
+            this.setCurrentModule();
```

Comment:
merge local `onClick` and the one from props

Reply:
It might be better to extract the `setCurrentModule` to the `onClick` handler of `File`, I noticed that when you click a directory we automatically open `src/index.js`. I assume that this happens because `this.setCurrentModule` is also called for directories, which resolves to an unknown module and then defaults to `src/index.js`.

Reply:
Yep, good catch!